gulpfile.js

```diff
@@ -21,7 +21,7 @@ gulp.task('copy', () =>
 
 // Process the Stylus files into CSS
 gulp.task('styles', () =>
-  gulp.src('static/src/styles/**/*.styl')
+  gulp.src('static/src/styles/main.styl')
```

Comment:
I believe the `changed` isn't doing anything in this case as by default it compares hashes of files, not modification dates. `main.styl` will always be different to `main.css`. I'm also not sure if it knows to change file extensions like that by default, either.